### PR TITLE
cleanup-update

### PR DESCRIPTION
**Pull request by pheobeayo**

### CODE REVIEW

**Comment by pheobeayo** [comment]
update has been done on the clean up

File: src/endpoints/achievements/verify_seniority.rs [hunk 1]
```diff
@@ -33,10 +34,10 @@ pub async fn handler(
 
     match execute_has_deployed_time(state.clone(), &query.addr).await {
         Ok(timestamp) => {
-            let dt = NaiveDateTime::from_timestamp_opt(timestamp as i64, 0).unwrap();
-            let current_time = Utc::now().naive_utc();
-            let duration = current_time - dt;
-            let days_passed = duration.num_days();
+            let timestamp_u64 = 1657113606;
```
Comment (Marchand-Nicolas) [inline src/endpoints/achievements/verify_seniority.rs]: Please don't use a static timestamp

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]
Need to update the code.
Also no need to open a new PR each time you have a change to do. Simply add more commits

**Comment by pheobeayo** [comment]
made a commit

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]
The "duration" should not be equal to the date_time, instead you should subtract the current time. Here's a piece of code that might help you:
```rs
      let current_time = Utc::now();
      let duration = current_time - date_time;
      let days_passed = duration.num_days();
```

**Review by Marchand-Nicolas — APPROVED** [review]
Well done Lgtm!

**Pull request merged**